sites/public/src/pages/account/edit.tsx

```diff
@@ -48,10 +76,12 @@ const Edit = () => {
   const [birthdateLoading, setBirthdateLoading] = useState(false)
   const [emailLoading, setEmailLoading] = useState(false)
   const [passwordLoading, setPasswordLoading] = useState(false)
+  const [user, setUser] = useState(null)
+  const [loading, setLoading] = useState(true)
 
-  const MIN_PASSWORD_LENGTH = 8
+  const MIN_PASSWORD_LENGTH = 12
```

Comment:
This would be a good value to have pulled in from environment variable in the future. Also, are all jurisdictions ok with increasing the min password length? If so do we have the right language listed everywhere? And also this would make the min password length different between public and partner accounts.

Reply:
Good q! We updated our password regex and security measures a bit ago to 12 characters, and one upper / one lower / one number / one special character, so this was a miss. BUT I do agree it should be pulled from one place.

Reply:
Updated both to 12 🫡